measures/2018/measures-schema.yaml

```diff
@@ -34,7 +34,7 @@ definitions:
         type: array
         items: { $ref: #/definitions/measureSets }
     required: [measureId, title, description, category, metricType, firstPerformanceYear, lastPerformanceYear]
-    oneOf:
+    anyOf:
```

Comment:
I don't think this makes any difference in practice since existing measure types are sufficiently different, but anyOf (must match one or more of) seems more precise than oneOf (must match exactly one of)